src/plugins/wresolver/wresolver.c

```diff
@@ -163,21 +164,21 @@ static void elektraResolveDir (resolverHandle * p, Key * warningsKey)
 		char buf[256];
 		FormatMessage (FORMAT_MESSAGE_FROM_SYSTEM, NULL, GetLastError (), MAKELANGID (LANG_NEUTRAL, SUBLANG_DEFAULT), buf, 256,
 			       NULL);
-		ELEKTRA_ADD_WARNINGF (ELEKTRA_WARNING_NOCWD, warningsKey, "GetCurrentDirectory failed: %s, defaulting to /", buf);
+		ELEKTRA_ADD_RESOURCE_WARNINGF (warningsKey, "GetCurrentDirectory failed: %s, defaulting to /", buf);
 		dir[0] = 0;
 	}
 	else if (dwRet > MAX_PATH)
 	{
-		ELEKTRA_ADD_WARNINGF (ELEKTRA_WARNING_NOCWD, warningsKey, "GetCurrentDirectory failed, buffer size too small, needed: %ld",
-				      dwRet);
+		// TODO: Solution? Compile with higher MAX_PATH?
```

Comment:
The solution is to submit a bug in the distribution's bug tracker to do that?

Reply:
I will change the text that either the user/dev can compile it himself or issue a bug so we do it for him in the main repo. If a single has a hilarious long path it might not be sensible to change the default.